components/org.wso2.carbon.identity.api.server.flow.management/org.wso2.carbon.identity.api.server.flow.management.v1/src/main/java/org/wso2/carbon/identity/api/server/flow/management/v1/utils/Utils.java

```diff
@@ -276,4 +298,155 @@ private static Map<String, Object> convertToMap(Object map) {
         return objectMapper.convertValue(map, new TypeReference<Map<String, Object>>() {
         });
     }
+
+    /**
+     * Checks whether a connector config is enabled.
+     *
+     * @param tenantDomain Tenant domain.
+     */
+    public boolean isFlowConfigEnabled(String tenantDomain, String connectorConfig) {
+
+        try {
+            IdentityGovernanceService identityGovernanceService =
+                    FlowMgtServiceHolder.getIdentityGovernanceService();
+            Property[] connectorConfigs = identityGovernanceService.getConfiguration(
+                    new String[]{connectorConfig}, tenantDomain);
+
+            if (connectorConfigs == null || connectorConfigs.length == 0) {
+                return false;
+            }
+
+            return Boolean.parseBoolean(connectorConfigs[0].getValue());
+
+        } catch (IdentityGovernanceException e) {
+            throw handleFlowMgtException(new FlowMgtClientException(
+                    ERROR_CODE_GET_GOVERNANCE_CONFIG.getCode(),
+                    ERROR_CODE_GET_GOVERNANCE_CONFIG.getMessage(),
+                    ERROR_CODE_GET_GOVERNANCE_CONFIG.getDescription(), e));
+        }
+    }
+
+    public List<IdentityProvider> getConnections() {
+
+        try {
+
+            IdpManager idpManager =
+                    FlowMgtServiceHolder.getIdpManager();
+            String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+            return idpManager.getIdPs(tenantDomain);
+
+        } catch (IdentityProviderManagementException e) {
+            throw handleFlowMgtException(new FlowMgtClientException(
+                    ERROR_CODE_GET_LOCAL_AUTHENTICATORS.getCode(),
+                    ERROR_CODE_GET_LOCAL_AUTHENTICATORS.getMessage(),
+                    ERROR_CODE_GET_LOCAL_AUTHENTICATORS.getDescription(), e));
+        }
+    }
+
+    public static void collectFlowData(
+            List<Step> steps,
+            Set<String> executors,
+            Set<String> identifiers,
+            Set<String> ids
+    ) {
```

Comment:
Normally we don't use this kind of formatting in backend. Can you fix this using the IDE formatting command.

Reply:
Check other places as well